formbricks/formbricks
# fix/logic on question update

## What does this PR do?



Fixes [FOR-980](https://linear.app/formbricks/issue/FOR-980/[bug]-logic-jumps-break-when-the-text-the-answer-option-changes)
Updates related logic values on multi select option change.


https://github.com/formbricks/formbricks/assets/42251569/ae69f06b-771c-4f22-85b5-9f69093ab313


## Type of change




## How should this be tested?



- Update MultiSelectSingleChoice choice label and see if the logic value updates as well and still performs the logic
- Update MultiSelectMultiChoice choice label and see if the logic value updates as well and still performs the logic

## Checklist
